pennylane/beta/devices/default_tensor.py

```diff
@@ -212,7 +212,7 @@ def _add_node(self, A, wires, name="UnnamedNode", key="state"):
         Returns:
             tn.Node: the newly created node
         """
-        name = "{}{}".format(name, wires.labels)
+        name = "{}{}".format(name, tuple(Wires(wires)))
```

Comment:
Was there a case where `wires` might not be a Wires object?

Reply:
Yes, whenever an indexed element from a `Wires` instance is passed to `_add_node`. This was not an issue before, as indexing gave back a new `Wires` instance. 
I have no overview over the tensor devices, so maybe this conversion can just be skipped by changing the expectations in beta/test_default_tensor.py , I am not sure about that.